### PR TITLE
adding debugging snippet try and catch

### PR DESCRIPTION
**Pull request by sannesofie**
### General Checks

- [ ] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] All CI checks pass
- [ ] all conflicts are resolved (if any)
- [ ] PR has a descriptive title
- [ ] PR has appropriate labels and milestones for easy identification
- [ ] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [ ] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [ ] the markdown source is formatted
- [ ] spelling and grammar is correct in all text
- [ ] The markdown looks correct when you preview the file
- [ ] all links and images work
- [ ] any embedded code snippets are formatted

### CODE REVIEW

File: vocabulary/javascript/debugging-try-and-catch.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Try and catch
+
+## Errors Will Happen
+
+If your program keeps crashing at a particular spot, try surrounding that code
+with try catch blocks to investigate the error object that gets returned.
+
+The `try` statement allows you to define a block of code to be tested for errors
```
Comment (arnochauveau) [inline vocabulary/javascript/debugging-try-and-catch.md:8]: You could use the `try-catch` statements for finding the errors, but if you don't. The errors will be logged automatically (with `console.error)` to the console by the browser. So it's not really that useful for that use case.

`try-catch`-es are usually used around things that can fail such as getting data from a server (the network could be disconnected, the server can be down, timeouts,...). You could then show a message to the user what went wrong.

errors you get when developing `!==` errors users get when using your software.

**Review by arnochauveau — CHANGES_REQUESTED** [review]
Hey @sannesofie, It seems you understand the technical side of the `try-catch` statement really well. However while reading your description, I've noticed some inaccuracies on **when** to use a `try-catch`

File: vocabulary/javascript/debugging-try-and-catch.md (hunk 1 above)
Comment (arnochauveau) [reply vocabulary/javascript/debugging-try-and-catch.md:8]: @sannesofie The most important thing is that you understood what I said. If that's not the case, let me know and I'll try and explain again. If you did understand, try changing line 5-6 to the thing you learned from the explanation. 🙂

Comment (sannesofie) [reply vocabulary/javascript/debugging-try-and-catch.md:8]: @arnochauveau I thought I understood it but I guess I did not LOL

Comment (arnochauveau) [reply vocabulary/javascript/debugging-try-and-catch.md:8]: Let's plan a call next week to go over it again 😁

Comment (sannesofie) [reply vocabulary/javascript/debugging-try-and-catch.md:8]: Thanks that would be great ;) I will message you on Discord

**Comment by gelilaa** [comment]
@sannesofie and @arnochauveau  can you give us some updates on this one? thank you!

**Comment by sannesofie** [comment]
> @sannesofie and @arnochauveau can you give us some updates on this one? thank you!

I have not had the time at all to look at this... will try and contact a coach when I find the time ;(

**Comment by sannesofie** [comment]
@gelilaa I had a call with Yassine and I made some changes, can you check this for me? Also I still keep getting this lint problem, but I have not made any changes in calendar in the home repository, is anyone else getting this fault

**Comment by gelilaa** [comment]
> @gelilaa I had a call with Yassine and I made some changes, can you check this for me? Also I still keep getting this lint problem, but I have not made any changes in calendar in the home repository, is anyone else getting this fault

@sannesofie  can you pull one time and push again?

**Pull request closed without merging**